tests/test_tokenizer.py

```diff
@@ -188,3 +186,18 @@ def test_errors(source, error_msg):
     with pytest.raises(TokenizerError) as excinfo:
         list(tokenizer.tokenize())
     assert str(excinfo.value) == error_msg
+
+
+@pytest.mark.parametrize(
+    "source",
+    [
+        "<turbo-frame></turbo-frame>",
+        "<turbo-frame />",
```

Comment:
I'd add more tests e.g. for `<foo-bar-baz>` and `<foo->`. I'd also test names which are not allowed e.g. `<->`, `<foo-->`, etc..

I'd also make sure we can't do anything weird when combined with member expressions, e.g. `<foo.->`